### PR TITLE
Shorten install instructions

### PR DESCRIPTION
**Pull request by Darth-Ness**
Pip3 works on Linux as well. I have tested this, as I was able to install the package this way.

### CODE REVIEW

**Comment by webdev03** [comment]
I think you need to change it to Linux not Linix

File: browser.py [hunk 1]
```diff
@@ -7,10 +7,9 @@
   import webview
 except:
   print("You need to have Pywebview installed to run FreeCat.")
-  print("On macOS run:")
+  print("On macOS or Linix run:")
```
Comment (webdev03) [inline browser.py:10]: Linux

**Review by webdev03 — CHANGES_REQUESTED** [review]
I think that's it

**Comment by Darth-Ness** [comment]
Your fork was merged with FreeCat. So It's to late now.

**Pull request closed without merging**